src/components/UserProfile/PropertyCarousel.js

```diff
@@ -0,0 +1,49 @@
+import Paper from "@mui/material/Paper";
+import Typography from '@mui/material/Typography';
+import Carousel from 'react-material-ui-carousel';
+import Stack from "@mui/material/Stack";
+
+import {paperProps} from './UserHeader.js';
```

Comment:
```suggestion
import {paperProps} from './UserProfile.js';
```
looks like paperProps is being defined in UserProfile.js but you imported it from UserHeader, not sure why it's not throwing an error

Reply:
woah what the heck? javascript is magic, great catch, thanks

Reply:
fixed, not actually using any props on Paper for now because the values used look really bad on the profile page